tests/python/sdk/test_users.py

```diff
@@ -11,26 +11,25 @@
 from cvat_sdk import Client, models
 from cvat_sdk.api_client import exceptions
 
-from shared.utils.config import USER_PASS
-
 
 class TestUserUsecases:
     @pytest.fixture(autouse=True)
     def setup(
         self,
-        changedb,  # force fixture call order to allow DB setup
         tmp_path: Path,
+        fxt_login: Tuple[Client, str],
         fxt_logger: Tuple[Logger, io.StringIO],
-        fxt_client: Client,
         fxt_stdout: io.StringIO,
-        admin_user: str,
     ):
         self.tmp_path = tmp_path
-        _, self.logger_stream = fxt_logger
-        self.client = fxt_client
+        logger, self.logger_stream = fxt_logger
         self.stdout = fxt_stdout
-        self.user = admin_user
-        self.client.login((self.user, USER_PASS))
+        self.client, self.user = fxt_login
+        self.client.logger = logger
+
+        api_client = self.client.api_client
+        for k in api_client.configuration.logger:
+            api_client.configuration.logger[k] = logger
```

Comment:
I'm not sure this solution is best, because api client loggers should be sub-loggers of the client logger. I suggest to rebuild the hierarchy here, starting from the client logger. Please check the Client constructor.